test/src/module/command/helpTest.c

```diff
@@ -584,6 +585,61 @@ testRun(void)
         strLstAddZ(argList, "repo-retention-archive");
         TEST_RESULT_VOID(testCfgLoad(argList), "help for backup command, repo-retention-archive option");
         TEST_RESULT_STR_Z(helpRender(helpData), optionHelp, "check admonition text");
+
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("multiple current values");
+
+        optionHelp = zNewFmt(
+            "%s - 'restore' command - 'repo-host' option help\n"
+            "\n"
+            "Repository host when operating remotely.\n"
+            "\n"
+            "When backing up and archiving to a locally mounted filesystem this setting is\n"
+            "not required.\n"
+            "\n"
+            "current:\n"
+            "  repo1: backup.example.net\n"
+            "  repo3: dr.test.org\n"
+            "\n"
+            "deprecated name: backup-host\n",
+            helpVersion);
+
+        argList = strLstNew();
+        strLstAddZ(argList, "/path/to/pgbackrest");
+        hrnCfgArgKeyRawZ(argList, cfgOptRepoHost, 1, "backup.example.net");
+        hrnCfgArgKeyRawZ(argList, cfgOptRepoHost, 3, "dr.test.org");
+        strLstAddZ(argList, "help");
+        strLstAddZ(argList, "restore");
+        strLstAddZ(argList, "repo-host");
+        TEST_RESULT_VOID(testCfgLoad(argList), "help for restore command, repo-host option");
+        TEST_RESULT_STR_Z(helpRender(helpData), optionHelp, "check text");
+
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("multiple current values (with one unset)");
```

Comment:
Not sure I understand the "unset" option here—just showing repo3's option?  Is the default for other repos supposed to be shown as well?

Reply:
Just to make sure gaps are rendered correctly, e.g. `repo1-path`, `repo3-path`.

Reply:
In other words, repo2 is skipped.

Reply:
Is it possible to set a global default per setting and override it on a repo-by-repo basis?  Is there a test exercising that if so? (Didn't seem to be in the new ones, which I'd expect if we're outputting for all of the known repos.)

Reply:
> s it possible to set a global default per setting and override it on a repo-by-repo basis?